package.json

```diff
@@ -60,7 +60,7 @@
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",
     "react-router-redux": "^4.0.8",
-    "react-scale-text": "^1.1.4",
+    "react-scale-text": "^1.2.1",
```

Comment:
Ok, the proptypes fix for this is in, can you update to 1.2.2 for react-scale-text?

Reply:
Fixed